### ISSUE TITLE
Text alternative for Helsinki logo, which acts as a link, should be "Go to the home page" 

### ISSUE THREAD

**Issue by kmhentto** [City-of-Helsinki/kerrokantasi-ui#782]
Palvelun etusivulle pääsee Helsingin kaupungin logoa klikkaamalla. Logo-linkki on kuvattu ruudunlukuohjelmille ” Helsinkilogo”. Teksti-vastineen perusteella ruudunlukuohjelman käyttäjä ei saa selville, mikä linkin todellinen tarkoitus on ja mihin linkki johtaa.
**Korjausehdotus**
Logolinkin tekstivastinetta tulee parantaa. Oleellisempaa on tietää, mihin linkki johtaa, joten tekstivastineena voidaan käyttää esimerkiksi ”Siirry etusivulle” -tekstiä.  Hox! Tämä pitää tehdä kaikissa kieliversiossa, ruotsiksi " Gå till hemsidan"  ja
englanniksi "Go to the home page".

**Comment by JoonasNislin** [comment]
Merged to master
